### DISCUSSION TITLE
pending booking delay email

### DISCUSSION THREAD

**Discussion by amsilva84** [liedekef/events-made-easy#335] [Q&A] [answered]
Hello 

I have set the "Set the number of days before reminder emails will be sent for pending bookings" to create a delay when the booking goes into pending status which it does right after someone clicks on the submit button for the form so the person doing the booking would receive an initial email. Is there another way of delaying this and removing the original email? did i miss something? 

A little background.  With a payment gateway selected it redirects to the payment page but if the window is closed the payment will not be completed so during the pending status of the booking I want to send the email pending reminder with the payment link but not right away during the initial booking

**Comment by liedekef** [comment, marked as answer]
Avoid sending the pending booking email totally: either set the subject or body for that email type to empty in the general EME settings, or - per event - assign an empty template to the subject/body.
After that, configure the pending reminder mail template, so then only that will be send out.

**Reply by amsilva84** [reply]
oh great, thank you! i missed that...
